### DISCUSSION TITLE
How do I use an icon from Material?

### DISCUSSION THREAD

**Discussion by AydinBiber-Dekker** [irihitech/Semi.Avalonia#282] [Q&A] [answered]
According to the README, "Material Design Icons" is credited so I'm assuming this theme includes their icons. But I can't find any information on how to use them.

**Comment by rabbitism** [comment, marked as answer]
Hello, we just use some material icon design in our built in control theme. We don't provide these icon for user usage.

**Reply by AydinBiber-Dekker** [reply]
That explains. Thanks!
